lib/medium-admin.php

```diff
@@ -1097,7 +1102,10 @@ private static function _handle_response($response) {
       $error_code = $response->get_error_code();
       error_log("WP ERROR: $message ($error_code)");
       if ($error_code == "http_request_failed" && strpos($message, "timed out") !== false) {
-        throw new Exception($message, -2); // our custom code for timeouts
+        throw new Exception($message, API_TIMEOUT);
+      }
+      if ($error_code == "http_request_failed" && strpos($message, "provisioned throughput for the table was exceeded") !== false) {
```

Comment:
We should use an HTTP error that implies retryable server error (is there such a thing?). Leaking such implementation details on public APIs is not a usual practice. 

Note that this is not a knock on your PR of course, which LGTM.